### ISSUE TITLE
Feature request : add swiss-french locale to the languages

### ISSUE THREAD

**Issue by spartid** [ubports/ubuntu-touch#1515]
### Description of the feature
When you install Ubuntu Touch, you can select the locale of your phone. Within the available languages you have French (France), but not French (Switzerland), which makes you get an AZERTY keyboard, and not the swiss-french one. You then have to select the swiss-french keyboard manually. An other annoying little bug is that when you manually type a number with the dialer app, it is displayed the french way (i. e. 06 28 29 10 19), and not the swiss way (079 492 28 19). You can still call anyone that way, but it is a little confusing at first.
Note : with the swiss indicator +41, the phone number is typed the correct way.

Would it be possible to add the French (Swizerland) locale ? It would be displayed in French : Français (Suisse), and would be linked to the swiss-french keyboard, so you would get the QWERTZ keyboard, and have the phone number "swissified" like the following one : 079 492 28 19.

**Comment by Flohack74** [comment]
Its a bit complicated. Is this locale available in Desktop Ubuntu ?

**Comment by ghost** [comment]
Hello, thanks for your answer ! What do you mean by "available" ?

If you mean that you can directly choose the locale when you install Ubuntu and select the installation language, then no. You have to select "French" (Français), and during the installation process you choose your geographical location (to set the clock) and then the correct keyboard layout. For swiss-french people, you find the layout under German --> French (Switzerland), because the QWERTZ layout is mainly used by German speaking people.

Thinking about this, would it be perhaps simpler to add the keyboard layout selection process during UT installation ?

**Comment by ghost** [comment]
> Well language and keyboard are 2 differen things. We can add a fr-ch keyboard, but there will be no need for a fr-ch language (text translation) right?

Ubuntu has no language fr-ch, you are right, because you simply can use french, and there is no need for a fr-ch language. What is different between desktop Ubuntu and UT is that with desktop Ubuntu, you can select your keyboard during the installation, whereas you can't with UT, and you'll end up with a fr-fr keyboard at the end of the process. You then have to manually select the correct keyboard.

As for the fr-ch keyboard, it is already there. There's still an [issue](https://github.com/ubports/keyboard-component/issues/100) with the dictionary, but I'm patient enough :) Thanks for your answer !

**Comment by Flohack74** [comment]
Ok so the feature request is here to select the keyboard earlier during the setup wizard?

**Comment by ghost** [comment]
Exactly, if this is possible. And if possible, to type phone numbers with the swiss pattern (for example 079 382 23 19) when the fr-ch keyboard is selected.

**Comment by Flohack74** [comment]
Thats then more an item for Lomiri, transferring the issue there...
